src/Contact.js

```diff
@@ -0,0 +1,279 @@
+import React from 'react';
```

Comment:
I like that you're initializing contact support through an email service. This is a great start and acts as an excellent placeholder. In the future, if you want to set up email support in your app I recommend Email.js.